libraries/WiFi/src/WiFiGeneric.h

```diff
@@ -31,6 +31,12 @@
 #include "esp_smartconfig.h"
 #include "wifi_provisioning/manager.h"
 
+#ifdef ENABLE_WPA2_AUTHENTICATION
```

Comment:
does it need to be disabled? Would it not work for all chips/cases? Having it guarded like that would require some way to enable this function.

Reply:
I had this guard in there because I wasn't sure if there was any shared code between the WiFi portions of the Arduino cores for ESP8266 and ESP32. AFAIK based on fairly recent testing, I don't think the WPA2 Enterprise authentication works for ESP8266, so I was thinking ahead to try and make it capable of being guarded easily. I can remove those, if you want me to.

Reply:
I see the client nowhere being used other than to set some certs that yu provide as arguments to the function. You should be able to safely skip those and add them to the sketch. BTW could you please provide an example of the use?

Reply:
I have a company profile on linkedin that can be used to identify me currently, a IBM is a computer of the guy that is stealing my identity.